src/AmbientSounds.Uwp/Views/ShellPage.xaml.cs

```diff
@@ -38,6 +38,27 @@ public ShellPage()
         // Set custom title bar.
         // Source: https://learn.microsoft.com/en-us/windows/apps/develop/title-bar?tabs=winui2#interactive-content 
         Window.Current.SetTitleBar(AppTitleBar);
+
+        Windows.UI.Core.Preview.SystemNavigationManagerPreview.GetForCurrentView().CloseRequested +=
+        async (sender, args) =>
+        {
+            args.Handled = true;
+
+            bool playing = App.Services.GetService<IMixMediaPlayerService>().PlaybackState == MediaPlaybackState.Playing ? true : false;
```

Comment:
I don't think the ternary operator is necessary here.

```suggestion
            bool playing = App.Services.GetService<IMixMediaPlayerService>().PlaybackState == MediaPlaybackState.Playing;
```